src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,14 @@
 
 public class MainApp {
 
+    private static final Machine[] machines = new Machine[]{new Truck(),
```

Comment:
As Nazar already mentioned: you need to create machines array directly in the main method, having this object as constant is redundant in this particular case